src/main/java/mate/controller/IndexController.java

```diff
@@ -2,10 +2,12 @@
 
 import java.io.IOException;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+@WebServlet(urlPatterns = "/index")
 public class IndexController extends HttpServlet {
     @Override
     public void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
Please, use one type of access modifier for all servlets for consistency. In all previous classes you used protected, so it's will be better use it here too 
```suggestion
    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```